src/main/resources/javascript/webSocket.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Created by chris1chang on 11/29/2016.
+ */
+
+//Establish the WebSocket connection and set up event handlers
+var webSocket = new WebSocket("ws://" + location.hostname + ":" + location.port + "/chat/");
```

Comment:
not certain, but it might need to be `/chat` and not `/chat/` to match `Path.Web.CHAT`